README.md

```diff
@@ -6,8 +6,10 @@ This module is experimental.
 Provides functionality to set up a RTCM base station to send RTCM correction streams through serial or i2c communication
 connected to a radio or Bluetooth transmitter. 
 This can be used for when there is no network connection or there are no public or private NTRIP mount points nearby.
-The base station should be set up with a clear view of the sky for the best accuracy. In the config, set your desired accuracy (1-5)
-and the time required to configure the base station.
+The base station should be set up with a clear view of the sky for the best fix. In the config, set your desired accuracy (meters)
```

Comment:
Does this mean any number in meters is ok for required_accuracy?

Reply:
yes